INSTALL.md

```diff
@@ -7,8 +7,10 @@ The simplest way to get the latest pandoc release is to use the installer.
   Download the latest installer
 </a>
 
-For alternative ways to install pandoc, see below
-under the heading for your operating system.
+For alternative ways to install pandoc, see below under the heading for
+your operating system. In cases where a statically linked executable is
+installed, the pandoc binary from this package cannot use lua filters that
+require external lua modules written in C.
```

Comment:
I'd suggest breaking this up.  First sentence in its own paragraph. Then a new paragraph:
```
Note: the statically linked pandoc binaries we provide cannot use Lua filters that
require external Lua modules written in C.  If you need to use such filters, choose
another installation method.
```

Reply:
I've separated that paragraph. I slightly tweaked wording as previously INSTALL.md only stated the Linux binary packages are statically linked, and it wasn't clear if this limitation applies to windows etc.